config/openapi-generator.json

```diff
@@ -0,0 +1,10 @@
+{
+    "apiNameSuffix": "Api",
+    "isGoSubmodule": true,
+    "disallowAdditionalPropertiesIfNotPresent": false,
+    "templateDir": "/local/templates",
+    "inlineSchemaOptions": {
+        "RESOLVE_INLINE_ENUMS": true
+    },
+    "enumClassPrefix": true
```

Comment:
Would it be possible to add an explicit `files` key for the model template? I know it works without it, but it might be clearer for people to understand how the model code is generated, if there's a record of what template goes to what files.

I'm actually not sure how it should be specified, maybe sth like
```json
"files:" {
  "model_simple.mustache": {
       "templateType": "model"
       "destinationFilename": "???"
  }
}
```

.. but I'm now reading through the oag code, and it seems that model_simple is a sub-template of the model template. So it might not even be possible to refer to a custom `model_simple` expiclitly.

Reply:
I'm not sure if that would work, but even if it does I'd be hesitant to make that config change in the scope of this PR, since nothing in this PR is dependent on custom templates.